app/views/network_router/add_interface_select.html.haml

```diff
@@ -0,0 +1,23 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "networkRouterFormController"}
```

Comment:
Please add `:novalidate => true` to the list of options for the `%form` here, so that the browser won't render that
annoying `"Please Select an Item from the List."` popup when you click `Cancel` in the add form. We do use
that in other angular forms as well.